src/polyhash/index.js

```diff
@@ -5,6 +5,7 @@ const turfBooleanDisjoint = require("@turf/boolean-disjoint").default;
 const turfIntersect = require("@turf/intersect").default;
 const turfHelpers = require("@turf/helpers");
 const turfMeta = require("@turf/meta");
+const turf = require("@turf/turf");
```

Comment:
Is it possible to import bboxPolygon module only? in a similar fashion to the above imports

Reply:
good point, will update